Beyond06-SOW.md

```diff
@@ -0,0 +1,346 @@
+| 11             |                   | Round table Q&A - job interview                                                                                                                                                                                                                                                           |                      |
+| 12             | 02.01 (last session)   | Final Project Demo and course retrospective                                                                                                                                                                                                                              |                      |
+
+
+
+## Final Grade
+
+* The Red Hat team will assign one of the following grades to each student which will be translated into the final grade by the professor. The grade is derived from:
+
+* Collaboration and engagement - To work together with others to achieve a common goal.
+* Execution process quality
+
+Collaboration and engagement - communication is almost everything. We expect to hear you. During classes, over Slack, in PRs and in GitHub issues. To be blunt about it - if by the end of the course, if none of the Red Hatters remember who you are - you will not get a good grade.
+
+Class attendance - while attendance will help you to get familiar with the material and necessary to engage and collaborate with the staff - it only won't affect the grade.
+
+* Grade 1 - hasn't any slack activity, without any responsiveness to his PRs.
+* Grade 2 - works alone. Does not discuss decisions made and issues with the mentors or his team mates.
+* Grade 3 - Responsible for closing his PRs quickly (uses office hours, if needed), slack activity inside the group and/or with the mentors, presenting in the demos.
+* Grade 4 - Review teammates PRs, the leader(s) of the group (the most engaged teammate(s) in others' tasks), helps others.
+
+Execution process quality - in this section we will measure the github activity (PRs) and the improvement in the technology stuck (frontend, backend, Data model, tests). We would like to see feature planning and the ability to learn new areas despite failures.
+
+* Grade 1 - not enough PRs, not involved in feature planning, was involved with only 1 layer from the technology stack(frontend, backend, Data model, tests).
+* Grade 2 - a few merged PRs ( minor PRs or 1-2 PR for a demo period) , involving only 2 layers of the technology stack(frontend, backend, DBs, tests).
+* Grade 3 - Wrote PRs involving at least 3 of the 4 layers in the technology stuck (frontend, backend, Data model, tests), part of feature planning, for each PR attached an issue.
+* Grade 4 - touching on all the technology stack (frontend, backend, Data model, tests), every week publish and merge at least 2 significant PRs, each PR includes tests, ability to learn new areas and subject on himself.
+
+## Grade Process 
```

Comment:
We already talk about the performance review above. We can probably just drop the res of this section as it does not correctly describe the actual grading process.

Reply:
Still seeing the section here...